docs/content/reference/video.md

```diff
@@ -27,62 +27,71 @@ snippet: archetypes/video_auto_frames
 Video support is new in Rerun, and has a few limitations:
 
 * [#7354](https://github.com/rerun-io/rerun/issues/7354): Only the MP4 container format is supported
-* [#7298](https://github.com/rerun-io/rerun/issues/7298): On native, only the AV1 codec is supported
 * [#7755](https://github.com/rerun-io/rerun/issues/7755): No AV1 support on Linux ARM
 * [#5181](https://github.com/rerun-io/rerun/issues/5181): There is no audio support
 * [#7594](https://github.com/rerun-io/rerun/issues/7594): HDR video is not supported
 * There is no video encoder in the Rerun SDK, so you need to create the video file yourself
-* A limited sets of codecs are supported on web (see below)
 
 ## Streaming video
 Rerun does not yet support streaming video support. For scenarios where you don't need live video, you can work around this limitation by logging many small `AssetVideo`s to the same Entity Path. See [#7484](https://github.com/rerun-io/rerun/issues/7484) for more.
 
 ## Codec support
+
+### Overview
+
+Codec support varies in the web & native viewer:
+
+|            | Browser | Native |
+| ---------- | ------- | ------ |
+| AV1        | ✅       | ✅      |
+| H.264/avc  | ✅       | ✅      |
+| H.265/hevc | 🚧       | ❌      |
```

Comment:
maybe 🚧 doesn't quite cut it. It's not like we're working on it actively (we can't - hevc support is a legal issue for mozilla). But 🟧 is worse

Reply:
trying 🔲 instead